src/main/docker/Dockerfile

```diff
@@ -7,18 +7,19 @@ LABEL vendor="OWASP"
 ARG APP_DIR=/opt/owasp/dependency-track/
 ARG DATA_DIR=/data/
 ARG USERNAME=dtrack
+ARG GID=0
 ARG WAR_FILENAME=dependency-track-apiserver.war
 
 # Create the directory where Dependency-Track will store its data (${DATA_DIR}) and the external library directory (${EXTLIB_DIR})
 # Create a user and assign home directory to a ${DATA_DIR}
 # Ensure UID 1000 & GID 1000 own all the needed directories
 RUN mkdir -p -m 770 ${DATA_DIR} \
     && adduser -D -h ${DATA_DIR} -u 1000 ${USERNAME} \
-    && chown -R ${USERNAME}:${USERNAME} ${DATA_DIR}
+    && chown -R ${USERNAME}:0 ${DATA_DIR} 
```

Comment:
Why not :${GID} - otherwise that new parameter isn't even used anywhere?

Reply:
The reason for this is to support arbitrary user IDs, such you can find in environments such as OpenShift.
You can read up [on it in the OpenShift Documentation](https://docs.openshift.com/container-platform/4.9/openshift_images/create-images.html#use-uid_create-images).

Regardless, since #1303 was merged that has this kind of behaviour already incorporated this PR is redundant, I think.